docs/src/refman/video.txt

```diff
@@ -189,6 +189,42 @@ See also: [al_get_video_frame]
 
 Since: 5.1.12
 
+## API: al_open_video_f
+
+```c
```

Comment:
No need for this, this gets added automatically by the documentation build system.